src/dnx.mono.managed/EntryPoint.cs

```diff
@@ -58,6 +58,19 @@ public static int Main(string[] arguments)
         // Set the default lib path to be next to the entry point location
         Environment.SetEnvironmentVariable(EnvironmentNames.DefaultLib, Path.GetDirectoryName(typeof(EntryPoint).Assembly.Location));
         Environment.SetEnvironmentVariable(EnvironmentNames.ConsoleHost, "1");
+        
+        var p = (int)Environment.OSVersion.Platform;
+        string os;
+        if ((p != 4) && (p != 6) && (p != 128))
```

Comment:
What is 128? And why we can't just use the [PlatformId enum](https://msdn.microsoft.com/en-us/library/3a8hyw88(v=vs.110).aspx)?